.github/workflows/integration.yml

```diff
@@ -37,6 +37,7 @@ env:
 jobs:
 
   changes:
+    if: github.event_name == 'pull_request'
     uses: ./.github/workflows/changes.yml
```

Comment:
My main concern here is that `changes` now takes ~8 minutes to execute on a recent run compared to 8 seconds on a prior run. Could this be improved by using the cache?

Reply:
Yes, it could improved using a cache I believe. The reason for that difference is because of the PR that moved the changes detection stuff into vdev, means we have to compile vdev now to run the command.

But, that is unrelated to the changes here. These change removes the 8 mins of doing that job from the merge queue case.